templates/mod/ssl.conf.erb

```diff
@@ -1,8 +1,16 @@
 <IfModule mod_ssl.c>
   SSLRandomSeed startup builtin
-  SSLRandomSeed startup file:/dev/urandom 512
   SSLRandomSeed connect builtin
+<% if @ssl_random_seeds and ! @ssl_random_seeds.empty? -%>
```

Comment:
So, removing this if statement - how do I set the default pre-existing template options? Putting them in ssl.pp?